packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -311,9 +307,13 @@ export interface KeyProps {
    *
    * Setting this to true adds a default statement which delegates key
    * access control completely to the identity's IAM policy (similar
-   * to how it works for other AWS resources).
+   * to how it works for other AWS resources). This matches the default behavior
+   * when creating KMS keys via the API or console.
    *
-   * @default false
+   * If the '@aws-cdk/aws-kms:defaultKeyPolicies' feature flag is set (the default for new projects),
+   * this flag will always be treated as 'true' and does not need to be explicitly set.
```

Comment:
I think this could do with an explanation of how you would achieve the same effect as this flag previously did in the "new" situation.

So the answer needs to be, if you *don't* want the IAM usage permissions, then you must now add a `policyDocument` with the principals who can use/admin.

While you're writing the example, you'll realize that this will be shit code to write and that it probably makes sense to add a helper to give admin permissions to a certain principal. Perhaps a subclass of `PolicyStatement`? 

Afterwards this also needs a `@deprecated` I think.

Reply:
A `grantAdmin()` method also would not be amiss.

Reply:
A `grantAdmin()` is in the roadmap; I flip-flopped on whether including it in this change would be too much at once, but I'm very happy to bring it in. I think it'll help a lot with the use case you're describing.